setup.py

```diff
@@ -118,7 +118,7 @@
         "azure-eventhub~=5.7.0",  # Used for EventHub E2E tests
         "azure-functions-durable",  # Used for Durable E2E tests
         "flask",
-        "fastapi",
+        "fastapi~=0.85.0",
```

Comment:
Can you please add a comment on where the modules are used?

```suggestion
        "fastapi~=0.85.0",  # Used for ASGIMiddleware test
```